frontend/src/upload.js

```diff
@@ -70,18 +70,32 @@ $(document).ready(function() {
   // on file upload by browse or drag & drop
   function onUpload(event) {
     event.preventDefault();
-    $('#page-one').attr('hidden', true);
-    $('#upload-error').attr('hidden', true);
-    $('#upload-progress').removeAttr('hidden');
-    //don't allow drag and drop when not on page-one
-    $('body').off('drop', onUpload);
-
     let file = '';
     if (event.type === 'drop') {
+      if (event.originalEvent.dataTransfer.files.length > 1){
+        document.l10n.formatValue('uploadPageMultipleFilesAlert')
+                     .then(str => {
+                       alert(str);
+                     });
+        return;
+      }
       file = event.originalEvent.dataTransfer.files[0];
     } else {
+      if (event.target.files.length > 1){
```

Comment:
I don't think this case will ever happen as long as we don't have `multiple` on the input element, but there's no harm in checking here too.